solutions/swb-reference/integration-tests/tests/isolated/sshKeys/get.test.ts

```diff
@@ -11,29 +12,74 @@ import { checkHttpError } from '../../../support/utils/utilities';
 describe('Get Key Pair negative tests', () => {
   const setup: Setup = new Setup();
   let adminSession: ClientSession;
+  //let adminUserId: string | undefined;
+  let project: { id: string };
 
-  beforeEach(async () => {
+  beforeAll(async () => {
     adminSession = await setup.getDefaultAdminSession();
+    //adminUserId = adminSession.getUserId();
+    const { data: costCenter } = await adminSession.resources.costCenters.create({
+      name: 'test cost center',
```

Comment:
use the random name generator like this: `name: randomTextGenerator.getFakeText('fakeCostCenterName')`